CONTRIBUTING.md

```diff
@@ -3,10 +3,12 @@
 **Pull requests are welcome**, but keep this in mind:
 
 - Projects must have a [free/libre](https://gnu.org/licenses/license-list.html) license. If they lack a license, get to the original maintainer and ask them to add a license.
+- Projects should not include a [Code of Conduct](https://en.wikipedia.org/wiki/Code_of_conduct). If a project includes one, get to the original maintainer and ask them to remove the code of conduct.
 - The scripts, add-ons or plugins must be useful in a project.
 - Follow the existing style.
 - Sort in alphabetical order.
-- Sort by Godot version.
-- Projects related to cryptocurrency or NFT integrations are not allowed, for reasons including (but not limited to) the risk of scams.
+- Sort by Redot version.
+~~- Projects related to cryptocurrency or NFT integrations are not allowed, for reasons including (but not limited to) the risk of scams.~~
```

Comment:
The decision to allow cryptocurrency/NFT integrations is not a decision that would be yours to make. Also line 12 is not needed.

Reply:
nfts are a form of expression and cocs are useless we should discouraged cocs